.github/workflows/python-poetry-release.yaml

```diff
@@ -79,14 +80,16 @@ jobs:
           poetry-version: ${{ inputs.poetry-version }}
           working-directory: ${{ inputs.working-directory }}
 
-      - name: Release to PyPI
-        uses: bakdata/ci-templates/actions/python-poetry-release@v1.18.2
+      - name: Create changelog
+        id: build_changelog
+        uses: bakdata/ci-templates/actions/changelog-generate@fix/bug-changelog
+        if: ${{ inputs.changelog == 'true'}}
```

Comment:
I think we should also check that `changelog-config` is also set in this case or does it also work without a config?

Reply:
it works without a config. The changelog action already have a default config